### PR TITLE
Folder structure improvements for QSG delivery

### PR DESCRIPTION
**Pull request by Kavindu-Dodan**
## Purpose
At the moment we have a complex folder structure for quick start guide. This PR contains a change that reduces its complexity.

Rather than more complex folder structure, our samples now follow below structure,

bin - Scripts required to run the guide
conf - Configuration file(s)
lib - Contains bundled executables (ex:- Jetty runner, backend service)
scenarios - Data necessary for different guide scenarios.
webApps - Contains all web apps required to run guides

## Goals

- Improve quality
- Reduce complexity
- Correcting bugs associated with complex structure

## Approach
Change the distribution build to match with required folder structure. Change of scripts (sh and bat) to accompany changes 

## Documentation
Documentation changes are necessary and will be updated

Related issued : #139

### CODE REVIEW

File: quick-start-guide/src/main/resources/bin/app-server.sh [hunk 1]
```diff
@@ -22,7 +22,12 @@ getProperty() {
    echo $PROP_VALUE
 }
 
-PROPERTY_FILE=server.properties
+# Relative directory paths
+CONF_DIR=../conf
+APP_DIR=../webApps
```
Comment (senthalan) [inline quick-start-guide/src/main/resources/bin/app-server.sh]: webapps

**Comment by Kavindu-Dodan** [comment]
@senthalan  suggested corrections were done.

File: quick-start-guide/pom.xml [hunk 2]
```diff
@@ -14,26 +14,42 @@
...
+            <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
+        </dependency>
     </dependencies>
 
+    <properties>
+        <qsg.app.location>./webapps</qsg.app.location>
+        <qsg.jar.location>./lib</qsg.jar.location>
+    </properties>
+
+
```
Comment (omindu) [inline quick-start-guide/pom.xml]: Remove extra line

Comment (Kavindu-Dodan) [reply quick-start-guide/pom.xml]: corrected :)

**Pull request merged**